src/test/java/org/folio/rest/api/TenantRefApiTests.java

```diff
@@ -456,6 +461,17 @@ public void migrationShouldNotCorrectFulfillmentPreferenceSpellingFromMigratedVe
       });
   }
 
+  @Test
+  public void keepReferenceData(TestContext context) {
```

Comment:
Doesn't it make sense to test all of the reference data types with corresponding module versions, not only Cancellation reasons?